app/views/users/show.html.erb

```diff
@@ -4,8 +4,35 @@
   <title>User Profile</title>
 </head>
 <body>
-  <h1>User Profile</h1>
-  <p>User ID: <%= @user.id %></p>
-  <!-- Display other user details here -->
+  <div class="user-container ">
+    <div class="users flex-spaced">
+      <div class="user-photo bordered">Photo</div>
+      <div class="details bordered flex-spaced">
+          <div class="user-name"><%= @user.name %></div>
+          <div class="post-nums">Number of posts: <%= @user.posts.count %></div>
+      </div>
+    </div>
+    <div class="bio bordered flex-c">
+      <p>bio</p>
+      <span><%= @user.bio %></span>
+    </div>
+    <div class="post-container .flex-column">
+    <% @user.posts.each_with_index do |post, index| %>
+      <div class="flex post-comment">
```

Comment:
- According to the project requirements, you are required to use the `three_recent_posts` method in your User model, please fix this 🙏